heka/sandbox/decoders/extract_executive_summary.lua

```diff
@@ -151,6 +165,50 @@ function process_message()
     msg.Fields[11].value    = cnts[3] -- yahoo
     msg.Fields[12].value    = cnts[4] -- other
 
+    msg.Fields[13].value = ""
+    msg.Fields[14].value = ""
```

Comment:
I don't really like using integer indices for Fields, but I'm not sure it's worthwhile to introduce more code complexity for the sake of readability.